src/Concordium/Client/Output.hs

```diff
@@ -1160,6 +1166,9 @@ showRatio r =
       den = denominator r
    in show num ++ " / " ++ show den ++ " (approx " ++ show (realToFrac r :: Double) ++ ")"
 
+showPartsPerHundredThousands :: Types.PartsPerHundredThousands -> String
```

Comment:
Please add docs describing the output format.

Reply:
Removed this. Using the `show` instance for `PartsPerHundredThousands`.